scripts/_prepare_deps/_deps.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Final
+
+arch_deps: Final[List[str]] = [
+    'libx11',
+    'mesa',
+    'libxrender',
+    'libxi',
+    'libxkbcommon-x11',
+    'wayland',
+]
+
+debian_deps: Final[List[str]] = [
+    'libx11-xcb-dev',
+    'libglu1-mesa-dev',
+    'libxrender-dev',
+    'libxi-dev',
+    'libxkbcommon-x11-dev',
+    'libwayland-dev',
+    'libxkbcommon-dev=1.5.0-1',
+]
```

Comment:
python-distutils should also be installed, also plain arch installation may lack some other packages like fontconfig2 or so, need to check that

Reply:
+ autoconf

Reply:
+ libtool

Reply:
btw, think twice before building on arch, your life will get miserable

Reply:
+ libxcb*dev

Reply:
just don't build qt from source, do your mental health a favor